core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -483,9 +483,10 @@
   {% include 'components/response_header_directive.html' %}
   {% include 'components/rule_editor_directive.html' %}
   {% include 'components/version_diff_visualization_directive.html' %}
-
+  
```

Comment:
Btw, if you're using Sublime Text, you can add something like this to your user settings (note the last of the options):

```
{
	"ensure_newline_at_eof_on_save": true,
	"highlight_line": true,
	"rulers":
	[
		80
	],
	"shift_tab_unindent": true,
	"spell_check": true,
	"tab_size": 4,
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true,
}
```